### ISSUE TITLE
cmdlets are broken because of the path issue

### ISSUE THREAD

**Issue by ChendrayanV** [ChendrayanV/PowerShell.Wiz.Utility#1]
cmdlets are broken because of the path issue

```PowerShell
Cannot find path 'C:\graphql\getCloudConfigurationRule.graphql' because it does not exist.
```

**Comment by ChendrayanV** [comment]
1. Issue is **fixed with the version 0.0.8** (Removed all old versions)
2. Test is in progress

**Comment by ChendrayanV** [comment]
closing the issue! Version 0.0.8 is stable!
